www/tutorial/app/responses/help.txt

```diff
@@ -0,0 +1,20 @@
+Commands:

```

Comment:
A note on this: The problem we found is that the xterm js library expect windows style line endings `\r\n`.  The rake task now generates files with the correct endings, but we may need to be vigilant to ensure that git doesn't mangle them.